admin/client/src/components/form-builder/index.js

```diff
@@ -103,9 +111,33 @@ export class FormBuilderComponent extends SilverStripeComponent {
     })
       .then(response => response.json())
       .then(json => {
+        const formSchema = Object.assign({}, { id: json.id, schema: json.schema });
```

Comment:
@flashbackzoo  Why do we have this extra `id` in there again? That's part of the schema payload itself, right? And in terms of UID for the state, we already use the schema URL.

![image](https://cloud.githubusercontent.com/assets/111025/14729843/5dfd804e-0895-11e6-9402-d33ebf28af5c.png)